charts/karavi-observability/values.yaml

```diff
@@ -157,7 +157,7 @@ karaviMetricsPowermax:
     # sidecarProxy.image: the container image used for the csm-authorization-sidecar.
     # Default value: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
     sidecarProxy:
-      image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
+      image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.13.0
```

Comment:
Authorization sidecar images can move to v2.1.0

Reply:
Done